src/enums/sizes.md

```diff
@@ -17,20 +17,8 @@ enum Foo {
     B,
 }
 
-#[repr(u32)]
-enum Bar {
-    A,  // 0
-    B = 10000,
-    C,  // 10001
-}
-
```

Comment:
I think it's useful to explain how enum variant values work, and students coming from C especially are likely to ask about it. If it's too much for this slide maybe move it to another slide or the speaker notes?

Reply:
It's definitely something the instructor can spend a minute on hand-coding — but it's important that the page works as a _slide_. So it has to be short and crisp and to the point. It was very overloaded before.